docs/modules/ROOT/pages/quarkus-azure-key-vault.adoc

```diff
@@ -5,17 +5,20 @@ include::./includes/attributes.adoc[]
 include::./includes/support.adoc[]
 
 https://azure.microsoft.com/en-us/products/key-vault[Azure Key Vault] is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, certificates, or cryptographic keys. 
-This extension allows you to create and retrieve secret from Azure Key Vault by injecting the following object inside your Quarkus application.
+This extension allows you to:
 
-* `com.azure.security.keyvault.secrets.SecretClient` 
-* `com.azure.security.keyvault.secrets.SecretAsyncClient` 
+* Create and retrieve secrets from Azure Key Vault by injecting the one or both of the following objects in a Quarkus application.
+    - `com.azure.security.keyvault.secrets.SecretClient` 
+    - `com.azure.security.keyvault.secrets.SecretAsyncClient` 
+    
+* Use secrets from the key vault directly inside your `application.properties`.
```

Comment:
not only in `application.properties`, but supported other ways that're support by SmallRye Config, e.g., `@ConfigProperty(name = "kv//secret1")`?

Reply:
Yes, we support both, the usage doc has sample for both.

Reply:
ah, I got your idea. But I think the secret can be used in `application.properties` also means it can be used as ConfigProperty.